docs: add doc for sendRawTransactionWithDetailedOutput (#1071)
matter-labs/zksync-web-era-docs: docs/build/api.md

}
```

### `zks_sendRawTransactionWithDetailedOutput`

> Executes a transaction and returns its hash as well as storage logs and events that would have been generated
> if the transaction has already been included into the block. The API has a similar behaviour to

Reviewer: ```suggestion
Executes a transaction and returns its hash, storage logs, and events that would have been generated
if the transaction had already been included in the block. The API has a similar behaviour to
```

`eth_sendRawTransaction` but with some extra data returned from it.

With this API Consumer apps can apply "optimistic" events in their applications instantly without having to wait for
zkSync block confirmation time.

> It’s expected that the optimistic logs of two uncommitted transaction that modify the same state will not have causal

Reviewer: `transactions
```suggestion
It’s expected that the optimistic logs of two uncommitted transactions that modify the same state will not have causal
```
